core/field_angle.js

```diff
@@ -100,6 +100,31 @@ Blockly.FieldAngle.WRAP = 360;
  */
 Blockly.FieldAngle.RADIUS = Blockly.FieldAngle.HALF - 1;
 
+
+/**
+ * Draws the border with the correct width.
```

Comment:
Update comment: 
"Adds degree symbol and recalculates width."

Does it actually save the computed width in a property?